action.yml

```
@@ -205,6 +253,39 @@ runs:
       path: .tmp-ansible-collection-checkout
       persist-credentials: false
       ref: ${{ inputs.git-checkout-ref }}
+      fetch-depth: >-
+        ${{
+          steps.compute-change-detection-coverage.outputs.change-detection-arg
+          && '0' || '1'
+        }}
+
+  - name: Log the next step action
+    if: >-
+      !inputs.collection-src-directory
+      && steps.compute-change-detection-coverage.outputs.change-detection-arg
+    run: >-
+      echo ▷ Create branches for change detection
+    shell: bash
+  - name: Create branches for change detection
+    if: >-
+      !inputs.collection-src-directory
+      && steps.compute-change-detection-coverage.outputs.change-detection-arg
+    run: |
+      # Create a branch for the current HEAD, which happens to be a merge commit
+      git checkout -b __action_branch_name
+
+      # Name the target branch
+      git branch ${{
+        github.event.pull_request.base.ref
+      }} --track origin/${{
+        github.event.pull_request.base.ref
+      }}
```

It's safer to quote inputs.
```suggestion
      git branch '${{
        github.event.pull_request.base.ref
      }}' --track 'origin/${{
        github.event.pull_request.base.ref
      }}'
```